.travis.yml

```diff
@@ -15,11 +15,11 @@ notifications:
 #  - julia -e 'Pkg.clone(pwd()); Pkg.build("DataStructures"); Pkg.test("DataStructures"; coverage=true)'
 after_success:
   # push coverage results to Coveralls
-  - julia -e 'cd(Pkg.dir("DataStructures")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
+  - julia -e 'import Pkg; cd(Pkg.dir("DataStructures")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
   # push coverage results to Codecov
-  - julia -e 'cd(Pkg.dir("DataStructures")); Pkg.add("Coverage"); using Coverage; Codecov.submit(Codecov.process_folder())'
+  - julia -e 'import Pkg; cd(Pkg.dir("DataStructures")); Pkg.add("Coverage"); using Coverage; Codecov.submit(Codecov.process_folder())'
 
   # Documenter auto-deploy
   # following `using` instruction is for triggering precompilation
-  - julia -e 'Pkg.add("Documenter")'
-  - julia -e 'cd(Pkg.dir("DataStructures", "docs")); include("make.jl")'
+  - julia -e 'import Pkg; Pkg.add("Documenter")'
```

Comment:
New way to do this in Julia 1.0 Documenter,
is to put a Project.toml in your docs  folder,
and do 
https://juliadocs.github.io/Documenter.jl/stable/man/hosting/#.travis.yml-Configuration-1

It would be better if we did it this way

Reply:
Indeed, I can add a commit for that, or should I put this in a separate PR ?

Reply:
commit to this PR is best.